### PR TITLE
Submitting rust2html

### PR DESCRIPTION
**Pull request by frostblooded**
# What
Homework 1 solution

# Who
FN 81621 - Nikolay Danailov

### CODE REVIEW

**Comment by frostblooded** [comment]
Бая commit-и станаха. Може да пробвам да ги squash-на ако искате?

**Comment by frostblooded** [comment]
Тези тестове дето фейлват не мисля че са от моя код.

**Review by dtrendafilov — COMMENTED** [review]
Sure, you can squash them a bit.
As for the PR - it is Homework 1, but it is also a rust2html tool, which is more important.

File: homework/81621/syntax_highlighter/src/lexer/tokenizer.rs [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+                chars.next();
+            } else if *character == '*' {
+                self.tokens.push(Token {
+                    r#type: TokenType::Multiply,
+                    string: Some(character.to_string()),
+                    number: None,
+                });
+
+                chars.next();
+            } else if *character == '/' {
```
Comment (dtrendafilov) [inline homework/81621/syntax_highlighter/src/lexer/tokenizer.rs:204]: It seems like half the code here is handling a single character operator or token, can you avoid duplicating all that in rust?

**Comment by dtrendafilov** [comment]
Close stale old edition pull requests.

**Pull request closed without merging**